docker/types/containers.py

```diff
@@ -392,6 +392,11 @@ def __init__(self, version, binds=None, port_bindings=None,
                 raise host_config_value_error("userns_mode", userns_mode)
             self['UsernsMode'] = userns_mode
 
+        if uts:
+            if uts != "host":
```

Comment:
Can we add a version check here as well? I believe this was only introduced recently.

Reply:
Hmm... as far as I am aware of, it seems that the `UTSMode` attribute was already present when the `HostConfig` structure itself was moved to within the `Container` API type; that's why I did not include a version check for that attribute.

The actual version when this attribute was introduced seems to be around 1.7, but in that version the `HostConfig` structure was in `runconfig`.

Reply:
yup, my bad, looks like it has existed for a long time now.